dll/win32/advapi32/wine/security.c

```diff
@@ -306,7 +304,7 @@ OpenProcessToken(HANDLE ProcessHandle,
                                 TokenHandle);
     if (!NT_SUCCESS(Status))
     {
-        ERR("NtOpenProcessToken failed! Status %08x.\n", Status);
+        TRACE("NtOpenProcessToken failed! Status %08x\n", Status);
```

Comment:
Demoting to `WARN` should be enough, as a first step at least.

Reply:
sure, I can do that.

Reply:
done